testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/OAuthClient.java

```diff
@@ -790,6 +793,27 @@ public AccessTokenResponse doClientCredentialsGrantAccessTokenRequest(String cli
         }
     }
 
+    public AccessTokenResponse doPreauthorizedTokenRequest(String preAuthorizedCode) throws Exception {
```

Comment:
Why it is not used in `PreAuthorizedGrantTest` ?

Reply:
Because I wanted to make the usage of the api and its parameters more explicit in the test. E.g. for testing the PreAuthorizedToken Feature, I build the requests in the test, for using the feature in other tests(e.g. the OID4VCI tests), I use the helper method here.

Reply:
Thank you for the clarification.